crates/repl/src/repl_sessions_ui.rs

```diff
@@ -198,97 +217,18 @@ impl Render for ReplSessionsPage {
-                    acc
-                });
-
-        for kernels in kernels_by_language.values_mut() {
-            kernels.sort_by_key(|a| a.name())
-        }
-
-        // Convert to a sorted Vec of tuples
-        let mut sorted_kernels: Vec<(SharedString, Vec<&KernelSpecification>)> =
-            kernels_by_language.into_iter().collect();
-        sorted_kernels.sort_by(|a, b| a.0.cmp(&b.0));
-
-        let kernels_available = v_flex()
-            .child(Label::new("Kernels available").size(LabelSize::Large))
-            .gap_2()
-            .child(
-                h_flex()
-                    .child(Label::new(
-                        "Defaults indicated with a checkmark. Learn how to change your default kernel in the ",
-                    ))
-                    .child(
-                        ButtonLike::new("configure-kernels")
-                            .style(ButtonStyle::Filled)
-                            // .size(ButtonSize::Compact)
-                            .layer(ElevationIndex::Surface)
-                            .child(Label::new("REPL documentation"))
-                            .child(Icon::new(IconName::Link))
-                            .on_click(move |_, cx| {
-                                cx.open_url(KERNEL_DOCS_URL)
-                            }),
-                    ),
-            )
-            .children(sorted_kernels.into_iter().map(|(language, specs)| {
-                let chosen_kernel = store.read(cx).kernelspec(&language, cx);
-
-                v_flex()
-                    .gap_1()
-                    .child(Label::new(language.clone()).weight(FontWeight::BOLD))
-                    .children(specs.into_iter().map(|spec| {
-                        let is_choice = if let Some(chosen_kernel) = &chosen_kernel {
-                            chosen_kernel == spec
-                        } else {
-                            false
-                        };
-
-                        let path = spec.path();
-
-                        ListItem::new(path.clone())
-                            .selectable(false)
-                            .tooltip({
-                                let path = path.clone();
-                                move |cx| Tooltip::text(path.clone(), cx)})
-                            .child(
-                                h_flex()
-                                    .gap_1()
-                                    .child(div().id(path.clone()).child(Label::new(spec.name())))
-                                    .when(is_choice, |el| {
-
-                                        let language = language.clone();
-
-                                        el.child(
-
-                                        div().id("check").tooltip(move |cx| Tooltip::text(format!("Default Kernel for {language}"), cx))
-                                            .child(Icon::new(IconName::Check)))}),
-                            )
-
-                    }))
-            }));
-
```

Comment:
Since we have a kernel options picker now, having this listing in the sessions UI is redundant (and hard to support given the reliance on an editor). Happy to drop this for now.